src/components/VideoCard/VideoCard.jsx

```diff
@@ -1,27 +1,54 @@
-import React from "react";
+import React, { useState } from "react";
 import { BsThreeDotsVertical } from "react-icons/bs";
 import { IoRemoveCircle } from "react-icons/io5";
 import { useLocation, useNavigate } from "react-router-dom";
 import { useVideo } from "../../context/VideoContext";
 import { addToHistory } from "../../utilities/History/add-history";
 import { removeFromHistory } from "../../utilities/History/remove-history";
+import { addToWatchLater } from "../../utilities/WatchLater/add-watch-later";
+import { removeFromWatchLater } from "../../utilities/WatchLater/remove-watch-later";
```

Comment:
You can make an index.js file in utilities and import all files from there

Reply:
Sure, will do that while refactoring the code.